TAF/testScenarios/integrationTest/UC_device_service/resource_command_addTags.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Documentation  profile
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource     TAF/testCaseModules/keywords/core-command/coreCommandAPI.robot
+Resource     TAF/testCaseModules/keywords/core-metedata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Setup Suite
+...                   AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Keywords  Run Teardown Keywords
```

Comment:
```suggestion
Suite Teardown  Run Teardown Keywords
```
There is only one keyword in the `Suite Teardown`; you don't need to use `Run Keywords`.